packages/thumbprint-react/components/ModalDefault/__snapshots__/test.jsx.snap

```diff
@@ -1216,6 +1216,256 @@ exports[`ModalDefaultFooter renders a sticky footer 1`] = `
         <Portal
           containerInfo={
             <body>
+              <div
```

Comment:
Hm, it's odd that this added a new modal to the snapshot. Could you dig into why?